### ISSUE TITLE
Is this the repo for the offical protoc-gen-prost from crates.io?

### ISSUE THREAD

**Issue by Maher4Ever** [Tuetuopay/protoc-gen-prost#1]
Hello,

The `protoc-gen-prost` has now been published to crates.io: https://crates.io/crates/protoc-gen-prost. I'm glad to see this happen! However, it has been published by another developer and the linked repository is different from this one. 

Since I've been following the development of this plugin for quite a while it was surprising to me, especially since the [release comment](https://github.com/tokio-rs/prost/pull/492#issuecomment-1079564170) in the PR linked to this repo. This raises the question: can the published crate be trusted?

**Comment by Tuetuopay** [comment]
No, it has not been done by me. But yes, the published crate can be trusted from what I could see of its source. I did not have the time to look whether it's feature complete with mine (esp. regarding crate generation) though. It is in essence the same as mine (thin wrapper around prost-build), but with a very different approach in the code (much much less straightforward).

What happened is, @neoeinstein and I were both doing plugins, and since I've been working on them on and off, he was more reactive with publishing than I were at the moment Prost 0.10 was cut out with the required API visibility changes. In some way, he sniped the crate. We talked on discord about joining forces, but it never led anywhere. I woke up one morning seeing the crate published, so I dropped the subject for the time being. I never took the time to go back to the PR and inform people about it.

I am not sure about the future of this repo. If @neoeinstein's one is feature equivalent, it'll likely be archived and I'll switch to the other one, and contribute to his.

Sorry for the rollercoaster, that's what happen when two people work on the same thing without being aware of the other. I noticed his work when one day I saw a message on discord talking about his plugin, once we both finished ours. So yeah, no promises here.
